monkey/monkey_island/cc/services/config_schema.py

```diff
@@ -1,4 +1,4 @@
-WARNING_SIGN = u" \u26A0"
+WARNING_SIGN = " \\u26A0"
```

Comment:
Need to make sure this is still correct - I believe the double \\ is bad. 
Also, I think it's preferable to just write ⚠ (as a character)...

Reply:
True, also it works as it were before, not sure why 2to3 changed it